syntax = "proto3";

package example;

service ResNet {
  rpc ClassifyImage (ImageMatrix) returns (ClassifyResult) {}
}

message ImageMatrix {
  repeated int32 image_matrix = 1;
}

message ClassifyResult {
  float category = 1;
}
